From b36d9389f384364610cadbb29bf4091d3dd4e513 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Mon, 1 May 2023 11:32:49 -0700
Subject: [PATCH] Find 'generate' under /usr/libexec instead of /lib

---
 netplan/cli/utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/netplan/cli/utils.py b/netplan/cli/utils.py
index 1b6f8bfdbe4c..aafa92983aca 100644
--- a/netplan/cli/utils.py
+++ b/netplan/cli/utils.py
@@ -37,7 +37,7 @@ config_errors = (ConfigurationError, LibNetplanException, RuntimeError)
 
 def get_generator_path():
     # FIXME: meson build uses proper libexecdir (+symlink)
-    return os.environ.get('NETPLAN_GENERATE_PATH', '/lib/netplan/generate')
+    return os.environ.get('NETPLAN_GENERATE_PATH', '/usr/libexec/netplan/generate')
 
 
 def is_nm_snap_enabled():
-- 
2.40.1

